app/models/gather_repository_maintenance_stats.rb

```diff
@@ -1,12 +1,33 @@
 class GatherRepositoryMaintenanceStats
+    def self.gather_bitbucket_stats(repository)
```

Comment:
Would some of this make more sense in the repository hosts instead of having `gather_bitbucket` and `gather_github`?

Reply:
I think that would make more sense now that I've gone through `repository_host` with this work. Then the API to get stats would just be `repository.gather_stats`. I'll shift this stuff over today.